vignettes/install-pksim.Rmd

```diff
@@ -29,24 +33,53 @@ Download the latest portable version of PK-Sim by clicking
 
 ## Enable using .dll from zip
 
-Windows Defender prevents applications downloaded from untrusted sources
-from being loaded. Before extracting the downloaded portable version,
-you have to unblock it.
+Windows Defender prevents applications downloaded from untrusted sources from
+being loaded. Before extracting the downloaded portable version, you have to
+unblock it.
 
 1.  Right click on the downloaded archive and select "Properties" (on
     Windows 11, you have to select "Show more options" first),
 2.  In the Properties-window, select the checkbox "Unblock".
 
 ## Restart R session
 
-Make sure that no instance of `{ospsuite}` is loaded. If so, restart the
-R session, as otherwise you might not be able to overwrite the existing
-portable version of PK-Sim.
+Make sure that no instance of `{ospsuite}` is loaded. If so, restart the R
+session, as otherwise you might not be able to overwrite the existing portable
+version of PK-Sim.
 
 ## Extract PK-Sim
 
-Extract the downloaded portable PK-Sim to
-`C:/Program Files/Open Systems Pharmacology/PK-Sim 12.0`.\
-Change the version number in the folder name to match the PK-Sim version
-you downloaded while keeping only the two digits. For example, if you
-downloaded PK-Sim 12.0.1, the folder name should be `PK-Sim 12.0`.
+Extract the downloaded portable PK-Sim to `C:/Program Files/Open Systems
+Pharmacology/PK-Sim 12.0`.  
+Change the version number in the folder name to match the PK-Sim version you
+downloaded while keeping only the two digits. For example, if you downloaded
+PK-Sim 12.0.1, the folder name should be `PK-Sim 12.0`.
+
+
+# Install and use PK-Sim in a project
+
+In the project's folder, run the following code that will download the latest
+PK-Sim "minimal" version and extract it to a folder called `PKSim`.
+
+```r
+download.file("https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/pksim_minimal.zip",
+              destfile = "pksim_minimal.zip"
+)
+unzip("pksim_minimal.zip", exdir = "PKSim")
+file.remove("pksim_minimal.zip")
+```
+
+Then, include the following code lines at the begining of the projects scripts:
+
+```r
+ospsuite::initPKSim(pksimFolderPath = "PKSim")
+```
+
+Alternatively, it is possible to add this line to a `.Rprofile` file in the
```

Comment:
"this line" is not clear, better "the line from above" or something... I was actually waiting for the code line to be specified here